Sources/Data/models/CustomFilterListSetting.swift

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import CoreData
+import Shared
+import os.log
+
+public final class CustomFilterListSetting: NSManagedObject, CRUD, Identifiable {
+  @MainActor @NSManaged public var uuid: String
+  @MainActor @NSManaged public var isEnabled: Bool
+  @MainActor @NSManaged public var externalURL: URL
```

Comment:
If you're going to add `@MainActor` to every function & property, just add it to the class itself instead